coremods-test/build.gradle

```diff
@@ -1,3 +1,6 @@
+// TODO I think this entire test project is unnecessary
+// is there a way we can move this to /src/test/java?
+
```

Comment:
No, this is the way that any modular test framework has to be laid out. With the 'tests' as a completely separate module/project.